packages/framer-motion/src/animation/legacy-popmotion/index.ts

```diff
@@ -157,12 +158,15 @@ export function animate<V = number>({
         driverControls.start()
     }
 
-    play()
+    autoplay && play()
```

Comment:
Heh, this was removed in the previous PR.

Reply:
It was sods law that I went from "hey we've never once used this in Motion" to "i don't need to actually play this animation"